common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -499,6 +500,15 @@ public void setShowNotificationsExplainer(boolean needToShow) {
         prefs.edit().putBoolean(PREFS_KEY_SHOW_NOTIFICATIONS_EXPLAINER, needToShow).apply();
     }
 
+    // Tax Categories
+
+    public boolean getHasDisplayedTaxCategoryExplainer() {
+        return prefs.getBoolean(PREFS_KEY_SHOW_TAX_CATEGORY_EXPLAINER, false);
+    }
+
+    public void setHasDisplayedTaxCategoryExplainer() {
+        prefs.edit().putBoolean(PREFS_KEY_SHOW_TAX_CATEGORY_EXPLAINER, true).apply();
+    }
```

Comment:
Is there a better place to put this?  This is only used by one activity, so it is not really a global setting.